app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -101,16 +105,18 @@
           {{ summary.text("The last day suppliers can apply.") }}
```

Comment:
<nitpick> 
In the extra bit of the message that warns about Saturdays and Sundays, we can tell at that point whether the date in question is going to be one of those days, right? 
Wouldn't it be more cool to have a contextual message say something like "Since {} is a Saturday, please publish questions and answers by {}"?  
</nitpick>